src/Components/SmartComponents/InsightsSystemRule/__snapshots__/InsightsSystemRule.test.js.snap

```diff
@@ -186,15 +186,15 @@ It means:
                       id=""
                       role="tooltip"
                     >
-                      <TooltipContent
+                      <Unknown
```

Comment:
I believe that this is not a problem, but I want to make sure that you have seen this TooltipContent is changed to Unknown on snapshot update.

Reply:
just checked it, it looks good in general the CVE detail page is a bit off so I have to go thought and check the UI components anyway but in another PR